O21.Game/Engine/GameRules.fs

```diff
@@ -31,7 +31,12 @@ let LevelHeight = 300
 [<Literal>]
 let TicksPerSecond = 10.0
 
+[<Literal>]
 let MaxPlayerVelocity = 3
+
+[<Literal>]
+let FreezeMovementTime = 6
```

Comment:
What is "freeze movement"? To me it seems like this is the time player sprite is supposed to stay still after death? Let's rename to `PostDeathFreezeTicks` then.